### PR TITLE
[8.0] pos_customer_display: backport enhancements from v10

### PR DESCRIPTION
**Pull request by alexis-via**
Display relevant info on LCD when Qty/price/discount is forced via the buttons of the POS frontend

### CODE REVIEW

File: pos_customer_display/static/src/js/customer_display.js [hunk 1]
```diff
@@ -21,13 +21,26 @@ openerp.pos_customer_display = function(instance){
...
+                var discount = line.get_discount();
+                if (discount) {
+                    price_unit = price_unit * (1.0 - (discount / 100.0));
+                    }
                 price_unit = price_unit.toFixed(currency_rounding);
-                var l21 = line.get_quantity_str_with_unit() + ' x ' + price_unit;
+                var qty = line.get_quantity();
+                // only display decimals when qty is not an integer
+                if (qty.toFixed(0) == qty) {
+                    qty = qty.toFixed(0);
```
Comment (hparfr) [inline pos_customer_display/static/src/js/customer_display.js:35]: I don't get it.

Comment (alexis-via) [reply pos_customer_display/static/src/js/customer_display.js:35]: The aim of this is to display 2 instead of 2.000, 3 instead of 3.000

Comment (hparfr) [reply pos_customer_display/static/src/js/customer_display.js:35]: Ok, I get it. Thx

**Review by hparfr — APPROVED** [review]

**Review by legalsylvain — APPROVED** [review]
:+1: Code review. No test. 

thanks for this improvment.

**Pull request merged**